Microsoft.DotNet.Interactive/Commands/DisplayValue.cs

```diff
@@ -5,11 +5,13 @@ namespace Microsoft.DotNet.Interactive.Commands
 {
     public class DisplayValue : KernelCommandBase
     {
-        public DisplayValue(FormattedValue formattedValue)
+        public DisplayValue(FormattedValue formattedValue, string id = null)
         {
             FormattedValue = formattedValue;
+            Id = id;
         }
 
         public FormattedValue FormattedValue { get; }
+        public string Id { get; }
```

Comment:
For consistency, `DisplayId` would be better.

Reply:
Although I think this identifies a "value", not a "display", so here an in `UpdateDisplayValue` maybe we should rename this to `ValueId`?